packages/appeals-service-api/src/lib/notify-validation.js

```diff
@@ -1,3 +1,13 @@
+function isValidAppealForSendStartEmailToLPAEmail(appeal) {
```

Comment:
Again, what if appeal is null or undefined?

Reply:
Probably ignore this comment. I can see that your tests are happy with empty responses.

Reply:
A current piece of tech debt with the way the appeal is allowed to be created. This is one of them 'it should never happen' situations, but as it's technically possible, it needs to be guarded against :(